tests/ui/specs/MediaPlayer/MediaPlayer-specs.js

```diff
@@ -6,92 +6,223 @@ describe('MediaPlayer', function () {
 
 				Page.spotlightDown();
 
-				expect(mediaPlayer.playButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.playButton.isFocused()).to.be.true();
 			});
 
 			it('should focus `previous` button on 5-way down, then left', function () {
-				expect(mediaPlayer.slider.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.slider.isFocused()).to.be.true();
 
 				Page.spotlightDown();
 				Page.spotlightLeft();
 
-				expect(mediaPlayer.previousButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.previousButton.isFocused()).to.be.true();
 			});
 
 			it('should focus back `play` button when navigating back to media controls ', function () {
-				expect(mediaPlayer.slider.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.slider.isFocused()).to.be.true();
 
 				Page.spotlightDown();
 				Page.spotlightLeft();
-				expect(mediaPlayer.previousButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.previousButton.isFocused()).to.be.true();
 				Page.spotlightLeft();
-				expect(mediaPlayer.shuffleButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.shuffleButton.isFocused()).to.be.true();
 				Page.spotlightLeft();
-				expect(mediaPlayer.repeatButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.repeatButton.isFocused()).to.be.true();
 				Page.spotlightUp();
 				Page.spotlightDown();
 
-				expect(mediaPlayer.playButton.isFocused()).to.be.true();
+				expect(mediaPlayerDefault.playButton.isFocused()).to.be.true();
 			});
 		});
 
 		describe('using pointer', function () {
+			it('should focus `next` button on hover', function () {
+				mediaPlayerDefault.hover('Next');
+
+				expect(mediaPlayerDefault.nextButton.isFocused()).to.be.true();
+			});
+
+			it('should focus `menu` button on hover', function () {
+				mediaPlayerDefault.hover('Menu');
+
+				expect(mediaPlayerDefault.menuButton.isFocused()).to.be.true();
+			});
+		});
+	});
+
+	describe('disabled', function () {
+		it('should have the slider knob focused', function () {
+			expect(mediaPlayerDefault.slider.isFocused()).to.be.true();
+			Page.spotlightDown();
+			Page.spotlightDown();
+
+			expect(mediaPlayerDisabled.slider.isFocused()).to.be.true();
+		});
+
+		it('should not play media on playButton click', function () {
+			expect(mediaPlayerDisabled.knob.getCSSProperty('left').value).to.equal('0px');
+
+			mediaPlayerDisabled.playButton.click();
+			Page.delay(1000);
+
+			expect(mediaPlayerDisabled.knob.getCSSProperty('left').value).to.equal('0px');
+		});
+
+		it('should not play next media on nextButton click', function () {
+			expect(mediaPlayerDisabled.source).to.equal('https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3');
+
+			mediaPlayerDisabled.nextButton.click();
+			Page.delay(1000);
+
+			expect(mediaPlayerDisabled.source).to.not.equal('https://sampleswap.org/mp3/artist/78152/HiatusManJBanner_Show-Stopper-160.mp3');
```

Comment:
Should this be equal to 'https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3' for clarity?